src/main/java/com/example/RaiseTechTask7remake/Controller.java

```diff
@@ -28,17 +22,14 @@ public List<String> getName() {
         return List.of("アバン", "ダイ");
     }
 
+
     @PostMapping("/postnames")

```

Comment:
REST APIのURLにはget/postなどの動詞は含めないのがベターですね。
扱うリソースの複数形が良いかと思います。
```suggestion
    @PostMapping("/names")

```

このあたりは`REST API URL`で調べてみるとよいです。